src/lv_demo_widgets/lv_demo_widgets.py

```diff
@@ -0,0 +1,575 @@
+#!/opt/bin/lv_micropython
+import lvgl as lv
+import time
+import SDL
+lv.init()
+
+LV_DEMO_WIDGETS_SLIDESHOW = 0
+LV_THEME_DEFAULT_COLOR_PRIMARY=lv.color_hex(0x01a2b1)
+LV_THEME_DEFAULT_COLOR_SECONDARY=lv.color_hex(0x44d1b6)
+
+LV_LED_BRIGHT_MIN = 120
+LV_LED_BRIGHT_MAX = 255
+
+SDL.init(w=480,h=320)
```

Comment:
Why is this example specific to SDL? Any problem running it on ESP32/STM32?

Reply:
No! No problem on the ESP32 except of screen size, see above